inventory/sample/group_vars/all/all.yml

```diff
@@ -2,6 +2,9 @@
 ## Directory where etcd data stored
 etcd_data_dir: /var/lib/etcd
 
+## etcd deployment mode. Choices are legacy or kubeadm
```

Comment:
I suggest to explicitly say that this cannot be changed one the cluster is installed.

nit: could we find another term than `legacy`?  or maybe a boolean `etcd_kubeadm_enabled`?

Reply:
good idea